mqtt-streaming/src/main/scala/akka/stream/alpakka/mqtt/streaming/scaladsl/MqttSession.scala

```diff
@@ -496,7 +499,9 @@ final class ActorMqttServerSession(settings: MqttSessionSettings)(implicit mat:
                               producerPacketRouter,
                               publisherPacketRouter,
                               unpublisherPacketRouter,
-                              settings)
+                              settings),
+            Props.empty,
+            true
```

Comment:
Named param?

Reply:
After last commit these lines changed so I don't thing this applies anymore, thanks.